src/components/home/elements/SendToken.vue

```diff
@@ -519,19 +520,36 @@ export default {
         wordList() {
             return this.seedPhrase.split(' ')
         },
+        noColdAddress() {
+            return Object.keys(this.coldAddresses).length === 0 && this.coldAddresses.constructor === Object
+        },
         isSubmitDisabled() {
-            return !(this.recipient && BigNumber(this.amount).isGreaterThan(0) && this.isValidRecipient(this.recipient) && (this.isValidAttachment || !this.attachment) && this.isAmountValid('hot') && this.address !== '')
+            return function(type) {
+                let recipient = type === 'hotWallet' ? this.recipient : this.coldRecipient
```

Comment:
simplify Line 528 - 530 to `let [recipient, attachment,  address] = type === 'hotWallet' ? [this.amount, this.recipient, this.attachment] : [this.coldAmount, this.coldRecipient, this.coldAttachment]` (remove coldAmount, coldAttachment, coldRecipient in refactor PR).